<ui version="4.0" >
 <class>CompletionConfigWidget</class>
 <widget class="QWidget" name="CompletionConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>853</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="sorting" >
     <property name="title" >
      <string>Sorting</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="sortingAlphabetical" >
          <property name="text" >
           <string>Alphabetical</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="sortingReverse" >
          <property name="text" >
           <string>Reverse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="sortingCaseSensitive" >
          <property name="text" >
           <string>Case Sensitive</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="sortingInheritanceDepth" >
          <property name="text" >
           <string>Inheritance Depth</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Order of groupings (select a grouping method to configure):</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="sortGroupingOrder" />
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="leftMargin" >
             <number>0</number>
            </property>
            <property name="topMargin" >
             <number>0</number>
            </property>
            <property name="rightMargin" >
             <number>0</number>
            </property>
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="groupingOrderUp" >
              <property name="text" >
               <string>^</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="groupingOrderDown" >
              <property name="text" >
               <string>\/</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filtering" >
     <property name="title" >
      <string>Filtering</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QCheckBox" name="filteringContextMatchOnly" >
        <property name="text" >
         <string>Suitable context matches only</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="filteringHideAttributes" >
        <property name="text" >
         <string>Hide completions with the following attributes:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="filteringAttributesList" />
      </item>
      <item>
       <widget class="KIntNumInput" name="filteringMaximumInheritanceDepth" >
        <property name="label" >
         <string>Maximum inheritance depth</string>
        </property>
        <property name="minimum" >
         <number>0</number>
        </property>
        <property name="maximum" >
         <number>20</number>
        </property>
        <property name="referencePoint" >
         <number>0</number>
        </property>
        <property name="specialValueText" >
         <string>Infinity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grouping" >
     <property name="title" >
      <string>Grouping</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="groupingMethods" >
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <column>
         <property name="text" >
          <string>Grouping method</string>
         </property>
        </column>
        <item>
         <property name="text" >
          <string>Scope type (local, namespace, global)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Scope (eg. per class)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Access type (public etc.)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Item type (function etc.)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="groupingUp" >
          <property name="text" >
           <string>^</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="groupingDown" >
          <property name="text" >
           <string>\/</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="stackedWidget" >
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="accessGrouping" >
           <layout class="QVBoxLayout" >
            <property name="spacing" >
             <number>8</number>
            </property>
            <property name="leftMargin" >
             <number>0</number>
            </property>
            <property name="topMargin" >
             <number>0</number>
            </property>
            <property name="rightMargin" >
             <number>0</number>
            </property>
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Access Grouping properties&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="accessConst" >
              <property name="text" >
               <string>Include const in grouping</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="accessStatic" >
              <property name="text" >
               <string>Include static in grouping</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="accessSignalSlot" >
              <property name="text" >
               <string>Include signals and slots in grouping</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2" >
           <layout class="QVBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="leftMargin" >
             <number>9</number>
            </property>
            <property name="topMargin" >
             <number>9</number>
            </property>
            <property name="rightMargin" >
             <number>9</number>
            </property>
            <property name="bottomMargin" >
             <number>9</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Item Grouping properties&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="itemTemplate" >
              <property name="text" >
               <string>Include templates in grouping</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="columnMerging" >
     <property name="title" >
      <string>Column Merging</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="columnMergeTree" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <column>
         <property name="text" >
          <string>Columns</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Merged</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Shown</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="columnUp" >
          <property name="text" >
           <string>^</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="columnDown" >
          <property name="text" >
           <string>\/</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>105</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>filteringHideAttributes</sender>
   <signal>toggled(bool)</signal>
   <receiver>filteringAttributesList</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>290</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel" >
     <x>307</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
